server/ServerFSM.cpp

```diff
@@ -190,6 +190,9 @@ namespace {
         return save_path.string();
     }
 
+    bool IsAutoSaveFile(const boost::filesystem::path& path)
```

Comment:
this seems misnamed...

Reply:
It checks if file exists and has extension from function `GetAutoSaveFileName`. How can I name it better?

Reply:
probly replace `Auto` with `Multiplayer` since it just checks that it exists and has the `.mps` extension, rather than anything specific to it being an autosave.

Reply:
Changed name.